cleverpush-example/src/main/java/com/example/cleverpush/App.java

```diff
@@ -1,13 +1,48 @@
 package com.example.cleverpush;
 
 import android.app.Application;
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Handler;
+
 import com.cleverpush.CleverPush;
+import com.cleverpush.NotificationOpenedResult;
+import com.cleverpush.listener.FinishActivity;
+import com.cleverpush.listener.NotificationOpenedCallbackListener;
+import com.cleverpush.listener.NotificationReceivedListener;
+import com.cleverpush.util.Logger;
 import com.google.firebase.FirebaseApp;
 
 public class App extends Application {
+    String url = "https://cleverpush.com/en/";
+
     @Override
     public void onCreate() {
         super.onCreate();
         FirebaseApp.initializeApp(this);
+
+        CleverPush.getInstance(this).enableDevelopmentMode();
+        CleverPush.getInstance(this).subscribe();
+        CleverPush.getInstance(this).init(
+                getString(R.string.channel_id),
+                (NotificationReceivedListener) result -> System.out.println("Received CleverPush Notification: " + result.getNotification().getTitle()),
+                result -> System.out.println("Opened CleverPush Notification: " + result.getNotification().getUrl()),
+                (NotificationOpenedCallbackListener) (result, finishActivity) -> {
```

Comment:
@cpKevinrodriguesnobre 

We need to use class because we need to inherit Activity class and use finish function from it```

`
public class AppActivity extends Activity {
}
`

So now in callback customer need to finish the activity like following
`appActivity.finish();`